cluster/examples/kubernetes/ceph/upgrade-from-v1.2-apply.yaml

```diff
@@ -220,4 +220,18 @@ rules:
   resources:
   - csidrivers
   verbs:
-  - create
\ No newline at end of file
+  - create
+---
+# Rook Ceph Operator Config ConfigMap
+# Use this ConfigMap to override Rook-Ceph Operator configurations.
```

Comment:
What if we just have a link to docs on this feature so they will get a more complete picture?

Reply:
There are no docs for this feature unless I have missed them?

Reply:
The only place to refer them currently is operator.yaml. Anyway, I doubt many users will read this yaml. They'll probably just follow the instruction to apply it.